src/tests/simulation/src/cluster.rs

```diff
@@ -960,15 +961,15 @@ impl KillOpts {
     /// Killing all kind of nodes.
     pub const ALL: Self = KillOpts {
         kill_rate: 1.0,
-        kill_meta: true,
+        kill_meta: false, // FIXME: make it true when multiple meta nodes are supported
```

Comment:
Ensure that the kill operation is not permitted for a single meta node. Otherwise, other components will shut down due to exceeding the maximum heartbeat interval.